core/conn/jdbc_type2/native/CSrvrConnect.cpp

```diff
@@ -268,10 +268,10 @@ void SRVR_CONNECT_HDL::addSrvrStmt(SRVR_STMT_HDL *pSrvrStmt,BOOL internalStmt)
         mapOfInternalSrvrStmt[pSrvrStmt->stmtName]= pSrvrStmt;  // +++ map error
     }
 
-
     //End of Soln. No.: 10-100202-7923
 
     count++;
+    //pSrvrStmt->myKey = count;
```

Comment:
Thanks for removing some unneeded comments. Is this particular one needed ? In general if it might be good to clean up the method being touched (remove sol numbers, commented out code if not needed or add more comments, remove any MFC related code etc)